### PR TITLE
feat: add solid helpers and solidstart example

### PR DESCRIPTION
**Pull request by ryoid**
Adapted from vue helpers.

### CODE REVIEW

**Comment by MaxLeiter** [comment]
hey @ryoid, thanks so much for the PR, we'd love to support Solid.

After some thought, would you mind moving the example to it's own repo? We're starting to add some bloat and affecting DX with the additional packages. I'd be happy to link to your repo from the docs

**Review by MaxLeiter — CHANGES_REQUESTED** [review]
thanks a lot @ryoid! Would you mind adding a changeset with `pnpm changeset` and addressing my small comments?

File: examples/solidstart-openai/postcss.config.cjs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
```
[the remark this answers is not in the document]
Comment (ryoid) [reply examples/solidstart-openai/postcss.config.cjs:1]: Yep, the example is from solidstart's `with-tailwind` template

File: examples/solidstart-openai/.gitignore [hunk 2]
```diff
@@ -0,0 +1,24 @@
+
+dist
+.solid
+.output
+.vercel
+.netlify
+netlify
+
```
[the remark this answers is not in the document]
Comment (ryoid) [reply examples/solidstart-openai/.gitignore:8]: done

File: .changeset/friendly-owls-think.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+'ai': minor
```
Comment (MaxLeiter) [inline .changeset/friendly-owls-think.md:2]: this should be a `patch`

**Review by MaxLeiter — CHANGES_REQUESTED** [review]

**Review by MaxLeiter — APPROVED** [review]

**Pull request merged**